Software_Carbon_Intensity/Appendix_A.md

```diff
@@ -1 +1,101 @@
-Add information here
+
+
+**What is SLI and SLO?**
+Reliability is the concept of a system having the quality of being trustworthy or of performing consistently well. Today most applications have a Service level objective (SLO) to facilitate monitoring. SLOs have typically 3 constraints
+1)	Service level indicator or SLI.
+2)	The target metric or Objective in percentile 
+3)	The observation window
+SLI + Objective +Observation window = SLO
+SLIs have an event, a success criterion, and specify where and how you record success or failure. It is specified as the proportion of events that were good. 
+
+Example: Percentage of Query requests that return an error code other than 5XX or a timeout measured on the Query Gateway Frontend
+
+SLIs exist to help engineering teams make better decisions. Your SLO performance is critical information to have when you’re making decisions about how hard and fast you can push your systems. SLOs are also important data points for other engineers when they’re making assumptions about their dependencies on your service or system. Lastly, your larger organization should use your SLIs and SLOs to make informed decisions about investment levels and about balancing reliability work against engineering velocity.
+Availability SLO example
+•	90% of HTTP requests as reported by the load balancer succeeded in the last 30 day window ( _Here 90% HTTP requests succeeded is the SLI , Return error code other than 5xx or timeout is the Objective and 30 days is the observation window_)
+
+Latency SLO examples:
+•	100% of requests in the last 5 minutes as measured at load balancer are served in less than 900ms
+•	99.99% of requests in the last 5 minutes as measured at load balancer are served in less than 500ms
+•	90% of requests in the last 5 minutes as measured at load balancer are served in less than 200ms
+
+**How organizations can re-define SRE considering SCI.**
+Software carbon intensity is a relative score between applications to be more carbon efficient, carbon aware and hardware efficient.
+
+SCI for applications (CI)= (Energy used by the application * Located based marginal carbon emissions) + Embodied Carbon per baseline
+
+Here the baseline is per API call, per additional user, per additional ML job etc.
+
+From an SRE principle and alignment then, organizations would like to track carbon emissions from their applications using the SCI score and raise alerts on breach, i.e. if it increases beyond a certain percentage . If an application has a SCI score of x, organizations would then track variance from this value and configure monitoring principles .
```

Comment:
So @srini1978 I think that "variance" has a very specific mathematical connotation and unless we want to refer to that here, I would suggest using a more generic term such as "variation."